verification/curator-service/ui/cypress/integration/components/ProfileTest.spec.ts

```diff
@@ -10,6 +10,7 @@ describe('Profile', function () {
 
         cy.contains('Alice Smith');
         cy.contains('alice@test.com');
-        cy.contains('reader, curator');
+        cy.contains('reader');
```

Comment:
If you want to test for it being in a chip, you could `get` rather than `contains`. No biggie, though.

Reply:
nah I don't want the test to break if we start showing "admin" in a card or whatever next, contains is fine.